CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+## [v1.1.0](https://github.com/bugcrowd/vulnerability-rating-taxonomy/compare/v1.0...v1.1) - 2017-04-13
+### Added
+- directory_listing_enabled
+- directory_listing_enabled.sensitive_data_exposure
+- directory_listing_enabled.non_sensitive_data_exposure
+- server_security_misconfiguration.path_traversal
+- cross_site_scripting_xss.reflected.self
+- cross_site_scripting_xss.reflected.non_self
+- cross_site_request_forgery_csrf.application_wide
+- cross_site_request_forgery_csrf.application_specific
+- cross_site_request_forgery_csrf.authenticated_action
+- cross_site_request_forgery_csrf.unauthenticated_action
+
+### Removed
+- poor_physical_security
+- social_engineering
+
+### Adjusted Priority
```

Comment:
Thoughts on renaming this section to `Changed` and enumerating what changed below (so we can group name/movements here as well)?
By doing the above, it adheres to http://keepachangelog.com/en/0.3.0/

For example, on the item below:
`- cross_site_scripting_xss.cookie_based priority changed from P4 to P5`

so to codify the syntax:
`- {current_id} {attribute} changed from {previous_value} to {current_value}`

and other examples:
```
- cross_site_scripting id changed from cross_site_scripting_xss to cross_site_scripting
- cross_site_scripting name changed from "Cross-Site Scripting (XSS)"  to "Cross-Site Scripting"
```

Thoughts?